bin/check-asg-instances-created.rb

```diff
@@ -78,9 +78,9 @@ def run
     time_utc_offset = time_now - time_now.utc_offset
 
     if !config[:warning_limit].nil?
-      warning = config[:warning_limit]
+      warning = config[:warning_limit].to_i
     elsif !config[:critical_limit].nil?
-      critical = config[:critical_limit]
+      critical = config[:critical_limit].to_i
```

Comment:
Might considering changing the if branch to also factor in blanks or other invalid values:

```ruby
[1] pry(main)> Integer ' '
ArgumentError: invalid value for Integer(): " "
from (pry):1:in `Integer'
[2] pry(main)> ' '.to_i
=> 0
[3] pry(main)> Integer 'boo'
ArgumentError: invalid value for Integer(): "boo"
from (pry):3:in `Integer'
[4] pry(main)> 'boo'.to_i
=> 0
```

That could give a bit more strict warnings on invalid configuration options.